apps/wekan/docker-compose.yml

```diff
@@ -0,0 +1,84 @@
+version: "3.7"
+
+services:
+  wekan:
+    image: ghcr.io/wekan/wekan:v7.26
+    container_name: wekan
+    environment:
+      # https://github.com/wekan/wekan/blob/main/docker-compose.yml
+      - MONGO_URL=mongodb://wekan-db:27017/wekan
+      - ROOT_URL=https://${APP_DOMAIN}
+      # https://github.com/wekan/wekan/wiki/Troubleshooting-Mail
+      - MAIL_URL=smtp://${MAIL_ADDRESS}:${MAIL_PASSWORD}@smtp.gmail.com:587
+      - MAIL_FROM=Wekan Notifications <noreply.wekan@${APP_DOMAIN}>
+    restart: unless-stopped
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${APP_DATA_DIR}/data/wekan:/data
+    ports:
+      - ${APP_PORT}:8080
+    depends_on:
+      - wekan-db
+    networks:
+      - tipi_main_network
+      - backend
```

Comment:
The backend network is not needed.

Reply:
> The backend network is not needed.

not needed if you have no problem exposing a database to the overlay network. this is not fine with me.

Reply:
backend network will remain as to discord communication, the network name will stay `backend` since docker creates `<app>_backend` and so tests ***could*** test for networks by name
f6e6a8e48ff1   runtipi_tipi_main_network   bridge    local
6a0b5be2200a   wekan_backend               bridge    local